auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -0,0 +1,281 @@
+                    class="open-action-btn"
+                    @click="open(item)"
+                  >
+                    Open
+                  </v-btn>
+                </span>
+
+                      <!-- More Actions Menu -->
+                <span class="more-actions mr-4">
+                  <v-menu
+                    offset-y left nudge-bottom="4"
+                    v-model="dropdown[index]"
+                  >
+                    <template v-slot:activator="{ on }">
+                      <v-btn
+                        small
+                        color="primary"
+                        min-height="2rem"
+                        class="more-actions-btn"
+                        v-on="on"
+                      >
+                        <v-icon>{{dropdown[index] ? 'mdi-menu-up' : 'mdi-menu-down'}}</v-icon>
+                      </v-btn>
+                    </template>
+                    <v-list>
+                      <v-list-item class="actions-dropdown_item">
+                        <v-list-item-subtitle @click="open(item)">
+                          <v-icon small>mdi-file-certificate-outline</v-icon>
+                          <span class="pl-1">Incorporate Now</span>
+                        </v-list-item-subtitle>
+                      </v-list-item>
+                      <v-list-item class="actions-dropdown_item">
+                        <v-list-item-subtitle @click="removeBusiness(item)">
+                          <v-icon small>mdi-delete</v-icon>
+                          <span class="pl-1">Remove</span>
+                        </v-list-item-subtitle>
+                      </v-list-item>
+                    </v-list>
+                  </v-menu>
+                </span>
+              </div>
+            </td>
+          </tr>
+        </template>
+        <template v-slot:no-data>
+        <span>Add an existing company, cooperative or society to manage it or<br> add a Name Request to complete your
+          incorporation or registration.</span>
+        </template>
+      </v-data-table>
+    </v-card>
+  </div>
+</template>
+
+<script lang='ts'>
+import { Account, CorpType, FilingTypes, LegalTypes, SessionStorageKeys } from '@/util/constants'
+import { Business, BusinessRequest } from '@/models/business'
+import { Component, Emit, Prop, Watch } from 'vue-property-decorator'
+import { Organization, RemoveBusinessPayload } from '@/models/Organization'
+import { mapActions, mapMutations, mapState } from 'vuex'
+import { Action } from 'vuex-class'
+import ConfigHelper from '@/util/config-helper'
+import Vue from 'vue'
+
+@Component({
+  computed: {
+    ...mapState('business', ['businesses']),
+    ...mapState('org', ['currentOrganization'])
+  },
+  methods: {
+    ...mapMutations('business', ['setCurrentBusiness']),
+    ...mapActions('business', ['createNamedBusiness'])
+  }
+})
+export default class AffiliatedEntityTable extends Vue {
+  @Prop({ default: () => [] })
+  readonly affiliatedEntities
```

Comment:
Does this have a type? If not, how about `Array<any>` for now?

Reply:
That works for now, will update!

This will change pending on the model updates almost 100% sure of that. Will apply the type now tho 👍

Reply:
FWIW, I think there is a lint rule about leaving variables to assume "any" type; explicitly typing things as "any" fixes the issue.

Reply:
Updated in next commit